src/constants/api.ts

```diff
@@ -0,0 +1,13 @@
+import EnvVars from './env';
```

Comment:
use with alias import

Reply:
I tried with alias but it is not able to access from it.